### ISSUE TITLE
Filename comparison is too strict.

### ISSUE THREAD

**Issue by jfdm** [david-christiansen/idris-code-highlighter#10]
If the base name of the specified file is not made strictly relative, idrishl can fail.

For example: 

+ `idrishl Data.Fuel` fails
+ `idrishl ./Data.Fuel` works

I believe the filter predicate used in `sortedHighlights` in `Main.idr` is not smart enough to work out that `Data.Fuel` and `./Data.Fuel` are the same file.

**Comment by david-christiansen** [comment]
Yep. There's unfortunately no good Idris library for normalizing file names right now, so this is a bit inconvenient.

**Comment by david-christiansen** [comment]
Though it looks like you're adding a bit of a hack that fixes it for some cases, at least :-)

**Comment by david-christiansen** [comment]
I don't think that this is fixed yet, because it really should use a filename normalization relative to the current working directory. But the situation is better.

**Comment by jfdm** [comment]
Yeah, I know its a hack, but without a good file path library, it is the best I could think off.

**Comment by david-christiansen** [comment]
Definitely an improvement for everyday usability! When I was doing the code samples for my dissertation, I just had scripts that set everything up right. Which is obviously ridiculous.

**Comment by jfdm** [comment]
at least with this fix, my next paper will more than likely have semantic highlighting!

**Comment by david-christiansen** [comment]
Nice!

**Comment by david-christiansen** [comment]
It would also be nice to get .lidr support working in it.
